Services/DataBaseService/Interfaces/ICommand.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace DataBaseService.Interfaces
+{
+    public interface ICommand<I>
+    {
+        void Execute(I Data);
+    }
+    public interface ICommand<in I, out T>
```

Comment:
Agree with Dasha, I and T look kinda weird

Reply:
Rename this